contexts/KiltDidV1_2.jsonld

```diff
@@ -0,0 +1,10 @@
+{
+  "@context": [
+    "https://w3id.org/security/v2",
+    {
+      "@protected": true,
+      "KiltPublishedCredentialCollectionV1": "https://github.com/KILTprotocol/spec-KiltPublishedCredentialCollectionV1",
+      "Sr25519VerificationKey2020": "https://github.com/KILTprotocol/spec-kilt-did#sr25519"
```

Comment:
both these types have context files associated with them, we could point to these instead?

Reply:
I think it's nicer for them to point at our own specification.

Reply:
Do you think it makes sense to also add the other key types to the context, and link them to the spec README? I feel that in this way we remove dependencies on third parties, although we still link to them from our README. It would anyway be faster to fix without the need to change the context itself.

Reply:
Could be problematic because they are defined in the security context which, as basically all the contexts here, have the `@protected` flag set keeping them from being overwritten. So that may actually trigger json-ld processor errors.